code/modules/mob/living/silicon/robot/dogborg/dog_sleeper_vr.dm

```diff
@@ -384,11 +384,11 @@
 		var/atom/target = pick(touchable_items)
 
 		//Handle the target being a mob
-		if(ishuman(target))
-			var/mob/living/carbon/human/T = target
+		if(isliving(target))
+			var/mob/living/T = target
 
 			//Mob is now dead
-			if(T.stat & DEAD)
+			if(T.stat == DEAD)
 				message_admins("[key_name(hound)] has digested [key_name(T)] as a dogborg. ([hound ? "<a href='?_src_=holder;adminplayerobservecoodjump=1;X=[hound.x];Y=[hound.y];Z=[hound.z]'>JMP</a>" : "null"])")
```

Comment:
This isn't a performance issue, but just as a favor to me (and other admins), can you ammend the message admins to be in an `if(ishuman))` block? Just the message_admins thing, because I don't want to get spammed as dogborgs eat every mouse like I know CERTAIN PEOPLE do.

Reply:
It didn't even occur to me that the change also meant logging was no longer restricted to ishuman stuff. Amended.